config/config.toml

```diff
@@ -90,14 +90,18 @@ path = '/var/lib/cnosdb/wal'
 #sync_interval = "0" # h, m, s
 
 [cache]
-## The maximum size of a mutable cache.
-#max_buffer_size = "128M" # 134,217,728 bytes
+# The maximum size of a mutable cache.
+max_buffer_size = "128M" # 134,217,728 bytes
 
-## The maximum amount of immutable caches.
-#max_immutable_number = 4
+# The maximum amount of immutable caches.
+max_immutable_number = 4
+
+# The partion number of memcache cache,default equal to cpu number
+partition = 16
+
+# The maximum size of database usage schema buffer
```

Comment:
- Does `database usage schema` means identity `database USAGE_SCHEMA`?
- The `size` is a property of the buffer of the `database usage schema`